packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -87,16 +88,31 @@ export function resolveStyleRules(
 
         resolveStyleRules(value, unstable_cssPriority, pseudo, media, combinedSupportQuery, result);
       } else if (property === 'animationName') {
-        // TODO: support RTL!
-        const keyframe = compileKeyframeRule(value);
-        const animationName = HASH_PREFIX + hashString(keyframe);
+        const animationNames = Array.isArray(value) ? value : [value];
+        let keyframeCSS = '';
+        let keyframeRtlCSS = '';
+        const names = [];
+        const namesRtl = [];
+        for (const val of animationNames) {
+          const keyframe = compileKeyframeRule(val);
+          const name = HASH_PREFIX + hashString(keyframe);
+          keyframeCSS += compileKeyframes(name, keyframe);
```

Comment:
```suggestion
          keyframeCSS += compileKeyframesToCSS(name, keyframe);
```

Nit, it will be nice to have more explicit naming 🐱